### DISCUSSION TITLE
node/p2p: consider deprecating TCP usage

### DISCUSSION THREAD

**Discussion by Wondertan** [celestiaorg/celestia-node#3039] [Ideas]
After #913 is merged. we will have both TCP and QUIC support, and we should consider dropping TCP support entirely.

**Comment by liamsi** [comment]
I'd recommend not prioritizing this. It might be good to make this a post-launch concern even.

**Comment by Wondertan** [comment]
No objections. 

Also note that this can be done in two simple stages:
* Removing 2 loc from default config to prevent nodes listening on TCP by default
  * New nodes won't run it 
  * Still able to connect to TCP nodes
  * Suggest removing TCP from configs to existing nodes
* Removing transport support entirely from libp2p constructor
  * Once we know there is no TCP nodes running(e.g. via Crawler)
